src/components/mobile-only/menu-footer/menu-footer.js

```diff
@@ -45,7 +48,14 @@ const MenuFooterContainer = props => {
 
   const handler = (option) => {
     if (activeOption === option) resetActiveOption();
-    else setActiveOption(option);
+    else {
+      if (isLandscapeMode && !isLandscapeSidebarCollapsed) {
```

Comment:
Here we can do it a bit simpler ;)

```suggestion
      isLandscapeMode && !isLandscapeSidebarCollapsed && collapseLandscapeSidebar();
      setActiveOption(option);
```

Reply:
this is removed in another PR